docs/non-normative/how-to-write-conventions/status-metrics.md

```diff
@@ -0,0 +1,87 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Status Metrics
+aliases: [status-metrics]
```

Comment:
curious why is this necessary?

Reply:
I think it's only needed (aliases) if you've moved a document so URL links aren't broken.

You can remove aliases